Doc/library/random.rst

```diff
@@ -303,6 +297,20 @@ be found in any statistics text.
 Alternative Generator
 ---------------------
 
+.. class:: Random([seed])
+
```

Comment:
I would like to leave this information in the intro section.

Please just make a stub entry for ``.. class:: Random([seed])`` with just a note that this class implements the default psuedo-random number generator.

Reply:
Like So ? That feel a bit short, but I'll trust you.